Lombiq.VueJs.Samples/Assets/Scripts/VueComponents/business-card.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <div class="card qr-card" v-if="!loading && cardId">
+        <div class="card-body" v-if="!error && currentCard">
+            <h5 class="full-name">{{ currentCard.BusinessCard.FirstName.Text }} {{ currentCard.BusinessCard.LastName.Text }}</h5>
+            <p v-if="currentCard.BusinessCard.Phone.Text">[[ Phone ]]: {{ currentCard.BusinessCard.Phone.Text }}</p>
+            <p v-if="currentCard.BusinessCard.Email.Text">[[ Email ]]: {{ currentCard.BusinessCard.Email.Text }}</p>
+        </div>
+        <div class="card-body" v-else-if="error">
+            <h5>[[ An error occurred. ]]</h5>
+            <p class="error-message">{{ error }}</p>
```

Comment:
The current class doesn't add any styling - let's use `message-error` from BS, and `px-1` for some horizontal padding:
```suggestion
            <p class="message-error px-1">{{ error }}</p>
```

Reply:
See screenshots here for the result: https://github.com/Lombiq/Orchard-Vue.js/pull/81#discussion_r1175036860.